content/docs/rfcs/64/README.md

```diff
@@ -0,0 +1,271 @@
+and at times of high network utilization these messages may not be relayed at all.
+
+### RLN Parameters
+
+For the Waku Network,
+the `epoch` is set to `1` second
+and the maximum number of messages published per `epoch` is limited to `1` per publisher.
+The `max_epoch_gap` is set to `20` seconds,
+meaning that validators MUST _reject_ messages with an `epoch` more than 20 seconds into the past or future compared to the validator's own clock.
+All nodes, validators and publishers,
+SHOULD use Network Time Protocol (NTP) to synchronize their own clocks,
+thereby ensuring valid timestamps for proof generation and validation.
+
+### Memberships
+
+Each publisher to the Waku Network SHOULD register an RLN membership
+with one of the RLN storage contracts
+moderated in the Sepolia registry contract with address [0xF1935b338321013f11068abCafC548A7B0db732C](https://sepolia.etherscan.io/address/0xF1935b338321013f11068abCafC548A7B0db732C#code).
+Initial memberships are registered in the Sepolia RLN storage contract with address [0x58322513A35a8f747AF5A385bA14C2AbE602AA59](https://sepolia.etherscan.io/address/0x58322513A35a8f747AF5A385bA14C2AbE602AA59#code).
+RLN membership setup and registration MUST follow [17/WAKU-RLN-RELAY](https://rfc.vac.dev/spec/17/#setup-and-registration),
+with the `staked_fund` set to `0`.
+In other words, the Waku Network does not use RLN staking. 
+
+### RLN Proofs
+
+Each RLN member MUST generate and attach an RLN proof to every published message
+as described in [17/WAKU-RLN-RELAY](https://rfc.vac.dev/spec/17/#publishing).
+Slashing is not implemented for the Waku Network.
+Instead, validators will penalise peers forwarding messages exceeding the rate limit
+as specified for [the rate-limiting validation mechanism](#rate-limit-exceeded).
+This incentivizes all nodes to validate RLN proofs
+and reject messages violating rate limits
+in order to continue participating in the network.
+
+## Network traffic
+
+All payload on the Waku Network MUST be encapsulated in a [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/)
+with rate limit proof extensions defined for [17/WAKU2-RLN-RELAY](https://rfc.vac.dev/spec/17/#payloads).
+Each message on the Waku Network SHOULD be validated by each relayer,
+according to the rules discussed under [message validation](#message-validation).
+
+### Message Attributes
+
+- The mandatory `payload` attribute MUST contain the message data payload as crafted by the application.
+- The mandatory `content_topic` attribute MUST specify a string identifier that can be used for content-based filtering. This is also crafted by the application. See [Autosharding](#autosharding) for more on the content topic format.
+- The optional `meta` attribute MAY be omitted. If present this will form part of the message uniqueness vector described in [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/).
+- The optional `version` attribute SHOULD be set to `0`. It MUST be interpreted as `0` if not present.
```

Comment:
Why a `SHOULD`? Shouldn't it be a `MAY` so they are free to use version 1 or 2?

Reply:
Mmm. A bit on the fence here, as I see what you mean. I think though the point is that we haven't fully defined v1 or v2 for the Waku Network (v1 is being deprecated in most instances as interworking with Status is not really necessary anymore and v2 is still experimental). SHOULD implies that apps could use v1 and v2, as long as they understand the current risks/limitations as we haven't specified either for the Waku Network.